packages/blocks/table/src/Table.tsx

```diff
@@ -112,15 +112,15 @@ export const Table: BlockComponent<AppProps> = ({
   accountId,
   aggregateEntities,
   aggregateEntityTypes,
-  createLinks,
   entityId,
   entityTypeId,
   entityTypes: schemas,
   entityTypeVersionId,
   initialState,
   linkedAggregations,
   updateEntities,
-  updateLinks,
+  updateLinkedAggregations,
+  createLinkedAggregations,
```

Comment:
Similar comment as elsewhere - this was previously alphabetical and now is something of a mix of alphabetical with some things grouped by target. I suggest one or the other

Reply:
Thanks, have addressed this as part of https://github.com/hashintel/hash/pull/497/commits/d6608c89e90520c182058f8725f08a2fbe150a73